### PR TITLE
switch to grayskull update

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -6,56 +6,59 @@ package:
...
+    - calphy_kernel --help
+    - calphy_run_averaging --help
+    - calphy_process_averaging --help
+    - calphy_run_integration --help
+    - calphy_process_integration --help
+  requires:
+    - pip
+    - lammps
+    - pyscal
+    - pylammpsmpi
```
Comment (jan-janssen) [inline recipe/meta.yaml:54]: @srmnitc I tried to automatically update the dependencies, but `lammps`, `pyscal` and `pylammpsmpi` are not included in the `setup.py`. I can include `lammps` in `pylammpsmpi` then `pyscal` would be the only issue. Do you plan to publish `pyscal` on `pypi`?

Comment (srmnitc) [reply recipe/meta.yaml:54]: Ah there is pyscal on pypi. Its just called `pyscal2`.

Comment (srmnitc) [reply recipe/meta.yaml:54]: https://pypi.org/project/pyscal2/

**Pull request closed without merging**